llumnix/backends/backend_interface.py

```diff
@@ -46,6 +46,7 @@ def add_request(self, request_id: str, server_info: ServerInfo,
         Args:
             request_id: Request ID.
             server_info: The information of the api server where the request come.
+            req_expected_step: The expected number of steps for the request.
```

Comment:
Briefly introduce what the `req_expected_step` parameter is intended for. For example, to implement pdd.